client/app/bundles/shared/components/Switch.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from 'react';
+import css from './Switch.scss';
+
+type Props = {
+  id?: string
+};
+
+type State = {
+  checked: boolean
+};
+
+export default class Switch extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = { checked: this.props.checked || false };
+  }
+
+  handleChange = () => {
+    this.setState({ checked: !this.state.checked })
+  };
+
+  render () {
+    const switchClassNames = `${css.switch}`;
```

Comment:
I don't think you need the back ticks here or in the following line. You should be able to pass in the values directly.